packages/sanity/src/structure/panes/document/inspectDialog/helpers.ts

```diff
@@ -40,7 +40,7 @@ export function selectElement(element: HTMLElement): void {
 }
 
 export function select(event: any): void {
-  selectElement(event.currentTarget)
```

Comment:
before, it was targetting the whole of the `code` component, now it targets more narrowly